.github/workflows/wheels.yml

```diff
@@ -33,17 +37,20 @@ jobs:
         config:
           # torch 1.13 is built with cuda 11.6 and 11.7
           - torch_version: "1.13.0+cu117"
+            torch_version_constraint: "1.13"
             cuda_version: "11.7.0"
             pip_index: https://download.pytorch.org/whl/cu117
             cuda_run_file: https://developer.download.nvidia.com/compute/cuda/11.7.1/local_installers/cuda_11.7.1_515.65.01_linux.run
 
           - torch_version: "1.13.0+cu116"
+            torch_version_constraint: "1.13"
             cuda_version: "11.6.2"
             pip_index: https://download.pytorch.org/whl/cu116
             cuda_run_file: https://developer.download.nvidia.com/compute/cuda/11.6.2/local_installers/cuda_11.6.2_510.47.03_linux.run
 
           # torch 1.12 is built with cuda 11.3 and 11.6, we ignore 11.3 for now
           - torch_version: "1.12.1+cu116"
+            torch_version_constraint: "1.12"
```

Comment:
nit: is this constraint in line with what is being used by `torch_version` just above? (I see `1.12.1`).

Also, wouldn't it be a bit more robust to make the `torch_version` be a function of `torch_version_constraint` + `cuda_version` ?